import/popups/qmldir

```diff
@@ -1,16 +1,26 @@
 module Sailfish.WebView.Popups
 plugin sailfishwebviewpopupsplugin
 singleton LocationSettings 1.0 LocationSettings.qml
-UserPrompt 1.0 UserPrompt.qml
```

Comment:
This change seems to affect some code in sailfish-browser. I hope you'll forgive me encroaching, but I just thought it'd be quicker to create the PR fix: https://github.com/sailfishos/sailfish-browser/pull/846

Reply:
thanks!